groovy/inductor_graph.groovy

```diff
@@ -0,0 +1,119 @@
+NODE_LABEL = 'images_build_inductor_dashboard'
+if ('NODE_LABEL' in params) {
+    echo "NODE_LABEL in params"
+    if (params.NODE_LABEL != '') {
+        NODE_LABEL = params.NODE_LABEL
+    }
+}
+echo "NODE_LABEL: $NODE_LABEL"
+
+node(NODE_LABEL){
+    stage("get scripts and target image") {
+        echo 'get scripts and target image......'
+        sh '''
+        #!/usr/bin/env bash
+        cd ${WORKSPACE}
+        rm -rf tmp
+        git clone https://github.com/chuanqi129/inductor-tools.git tmp
+        old_container=`docker ps |grep inductor_${target_tag}|awk '{print $1}'`
+        if [ -n "${old_container}" ]; then
+            docker stop $old_container
+            docker rm $old_container
+            docker container prune -f
+        fi
+        docker pull ccr-registry.caas.intel.com/pytorch/pt_inductor:${target_tag}
+        '''
+    }   
+    stage("loggin container & cp scripts & collect graph"){
+        echo 'running......'
+        sh '''
+        #!/usr/bin/env bash
+        docker run -tid --name inductor_${target_tag} --privileged --env https_proxy=${https_proxy} --env http_proxy=${http_proxy} --net host  --shm-size 1G -v ${WORKSPACE}/graph/${target_tag}:/workspace/pytorch/${target_tag} ccr-registry.caas.intel.com/pytorch/pt_inductor:${target_tag}
+        echo '#!/usr/bin/bash
+        # set -x
+        SUITE=$1
+        MODEL=$2
+        CHANNELS=$3
+        BS=$4
+        TARGET_TAG=$5
+
+        MODEL_LIST=($(echo "${MODEL}" |sed '"'"'s/,/ /g'"'"'))
+
+        for SINGLE_MODEL in ${MODEL_LIST[@]}
```

Comment:
Can we move out the `for` loop, and do the `for` loop for `docker exec ...`? I feel this method is more readable. You can decide which is a better method